src/_base/helm/app/values.yaml.twig

```diff
@@ -3,8 +3,9 @@ appVersion: {{ @('app.version') | json_encode | raw }}
 feature: {{ to_nice_yaml(@('helm.feature'), 2, 2) | raw }}
 
 ingress:
+  annotations: {{ to_nice_yaml(@('pipeline.base.ingress.annotations'), 2, 4) | raw }}
   target_service: {{ @('services.varnish.enabled') ? 'varnish' : 'webapp' }}
-  type: "standard" # standard or istio
+  type: {{ @('pipeline.base.istio.enabled') ? 'istio' : 'standard' }}
```

Comment:
```suggestion
  type: {{ @('pipeline.base.ingress.type') }}
```
since we already have it in a project and go harness (tho app.ingress latter), and the keys follow the same pattern in values and workspace

Reply:
could also go the whole length and do the full ingress as well by migrating the target_service to workspace.yml so it'd be
```
ingress: {{ to_nice_yaml(@('pipeline.base.ingress'), 2, 2) | raw }}
```

Reply:
Done :+1: